src/notification/toast.component.ts

```diff
@@ -64,6 +65,10 @@ export class Toast extends Notification implements OnInit {
 	@HostBinding("class.bx--toast-notification--success") get isSuccess() { return this.notificationObj["type"] === "success"; }
 	@HostBinding("class.bx--toast-notification--warning") get isWarning() { return this.notificationObj["type"] === "warning"; }
 
+	get isShowClose() {
```

Comment:
Same comment as for notification here ... we may even be able to just use the getter from notification

Reply:
Good point!